jenkins/scripts/bare_metal_lab/templates/bmhosts_crs.yaml.j2

```diff
@@ -20,14 +20,14 @@ data:
 apiVersion: metal3.io/v1alpha1
 kind: BareMetalHost
 metadata:
-  name: eselda13u31s03.est.lan
+  name: eselda13u31s05.est.lan
 spec:
   online: true
-  bootMACAddress: b4:b5:2f:6d:89:d8
+  bootMACAddress: 80:c1:6e:7a:5a:a8
   bootMode: legacy
   bmc:
-    address: ilo4://192.168.1.12
-    credentialsName: bml-ilo-login-secret-03
+    address: ilo4://192.168.1.14
```

Comment:
this needs  to be changed: https://github.com/metal3-io/project-infra/blob/main/jenkins/scripts/bare_metal_lab/deploy-lab.yaml#L9

Reply:
Thank you for noticing. Done.